services/ui-src/src/utils/reports/entities.ts

```diff
@@ -108,7 +108,7 @@ export const renderTargetPopulationFields = (
         id: field.id,
         label: field.isRequired
           ? field.transitionBenchmarks_targetPopulationName
-          : `Other: ${field.transitionBenchmarks_targetPopulationName}`,
+          : `Other: {${field.transitionBenchmarks_targetPopulationName}}`,
```

Comment:
I'm not sure what this change does 🤔 
I tested with and without and didn't see a visual difference, though I would expect this change to add curly braces to the view